filepicker/filepicker.go

```diff
@@ -17,6 +17,9 @@ import (
 	"github.com/dustin/go-humanize"
 )
 
+// DefaultModTimeLayout is the default layout used to format modification time for a given file.
+const DefaultModTimeLayout = "Jan _2 15:04"
```

Comment:
Any particular reason to export this?

Reply:
No particular reason expect to have it easily accessible in the documentation.

We could make it private if you prefer.

Reply:
Ah, well that's good enough reason to keep it public.